### ISSUE TITLE
Develop test case: Bookmark folders can be copied via context menu.

### ISSUE THREAD

**Issue by tracywalker** [mozilla/iris#126]
https://testrail.stage.mozaws.net/index.php?/cases/view/4151

**Comment by IonutBudeanu** [comment]
Fixed in issue_126 branch.

Please review.

Thank You

**Comment by tracywalker** [comment]
Ionut, this works fine on Windows and Mac.  However, on Linux is fails to perform the DnD on line 59, thus the subsequent assert, lines 61-62, fails.

**Comment by IonutBudeanu** [comment]
Hi Tracy,

I solved the problem with a UI_DELAY before DnD. It seems that Linux needs some time before performing a drag & drop action.

Please take a look. Thanks

**Comment by tracywalker** [comment]
Interestingly, a delay works. There is some sort of focus or coordinate issue that occurs immediately after entering text to search the sidebar.  I discovered this issue can also be fixed by select_location_bar() instead of a UI_DELAY. We saw a similar problem in a past test, As such something should be done to make DnD generally more robust. I'll file an issue and investigate today.

For now we'll leave the DELAY in here; code approved.
